### PR TITLE
SHA-2361 : [Community] Can't list users in admin tools

### PR DESCRIPTION
**Pull request by alexbalmus**
- Fixed the problem by having how-authorization-status false by default (share-config.xml) and rely on the share-config-custom.xml that we ship with the dist. zip to override that.

### CODE REVIEW

File: share/src/main/resources/alfresco/share-config.xml [hunk 1]
```diff
@@ -129,7 +129,7 @@
          <!-- minimum length for username and password -->
          <username-min-length>2</username-min-length>
          <password-min-length>3</password-min-length>
-         <show-authorization-status>true</show-authorization-status>
```
Comment (alexbalmus) [inline share/src/main/resources/alfresco/share-config.xml]: Have this false by default.

File: packaging/distribution/pom.xml [hunk 2]
```diff
@@ -83,25 +83,6 @@
             <plugin>
                 <artifactId>maven-dependency-plugin</artifactId>
                 <executions>
-                    <!-- Extract share-config-custom.xml -->
```
Comment (alexbalmus) [inline packaging/distribution/pom.xml]: Forget about downloading the alfresco-installer-resources jar and extracting the share-config-custom.xml from there. Just store it alongside the resources (see below).

File: packaging/distribution/src/main/resources/web-extension-samples/share-config-custom.xml [hunk 3]
```diff
@@ -0,0 +1,527 @@
+<alfresco-config>
```
Comment (alexbalmus) [inline packaging/distribution/src/main/resources/web-extension-samples/share-config-custom.xml]: We'll just store this along side the web-extension-samples files.

File: packaging/distribution/src/main/resources/web-extension-samples/share-config-custom.xml [hunk 4]
```diff
@@ -0,0 +1,527 @@
...
+            <name>Alfresco Public API - user access</name>
+            <description>Access to Alfresco Repository Public API that require user authentication.
+                         This makes use of the authentication that is provided by parent 'alfresco' endpoint.</description>
+            <connector-id>alfresco</connector-id>
+            <endpoint-url>http://localhost:8080/alfresco/api</endpoint-url>
+            <identity>user</identity>
+         </endpoint>
+      </remote>
+   </config>
+
```
Comment (alexbalmus) [inline packaging/distribution/src/main/resources/web-extension-samples/share-config-custom.xml]: Here's the override for the show-authorization-status:

File: share/src/main/resources/alfresco/share-config.xml (hunk 1 above)
Comment (astrachan) [reply share/src/main/resources/alfresco/share-config.xml]: I would say it’s OK to remove the setting - we should however call-out the change in any subsequent release notes and point to the docs or explicitly state where they can re-enable

**Pull request merged**